ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/KVUtilsConfigSpec.scala

```diff
@@ -54,20 +54,22 @@ class KVUtilsConfigSpec extends AnyWordSpec with Matchers {
 
     "check generation" in KVTest.runTest {
       for {
-        logEntry <- submitConfig(
+        result <- preExecuteConfig(
```

Comment:
Nit:

```suggestion
        result1 <- preExecuteConfig(
```

And in other occurrences.